aws-services/reference/supported-topologies.hbs.md

```diff
@@ -41,16 +41,28 @@ For instructions for these tasks, see the
 After completing configuration in AWS, you must configure your `aws-services-values.yaml` file using
 the following values when installing the package:
 
-```yaml
-postgresql:
-  enabled: true
-  region: "REGION"
-  infrastructure:
-    subnet_group:
-      name: "SUBNET-GROUP-NAME"
-    security_groups:
-      - id: "SECURITY-GROUP-ID"
-```
+* PostgreSQL:
+    ```yaml
+    postgresql:
+      enabled: true
+      region: "REGION"
+      infrastructure:
+        subnet_group:
+          name: "SUBNET-GROUP-NAME"
+        security_groups:
+          - id: "SECURITY-GROUP-ID"
+    ```
+* MySQL:
+    ```yaml
+    mysql:
+      enabled: true
+      region: "REGION"
+      infrastructure:
+        subnet_group:
+          name: "SUBNET-GROUP-NAME"
+        security_groups:
+          - id: "SECURITY-GROUP-ID"
+    ```
```

Comment:
I suppose that if one of them, either MySQL or Postgres, is disabled earlier at installation, you don't need to provide values for it?

Reply:
they're not two separate phases - you provide values at installation, and that's how you enable services